devops/windows/base-docker-wsl/docker-stack.yml

```diff
@@ -0,0 +1,268 @@
+version: '3.8'
+
+networks:
+  backend:
+    name: backend
+    driver: overlay
+    attachable: true
+  frontend:
+    name: frontend
+    driver: overlay
+    attachable: true
+
+services:
+
+  postgresql:
+    image: skunkworks2:5000/v2/bitnami/postgresql:15.4.0
+    networks:
+    - backend
+    ports:
+    - published: 5432
+      target: 5432
+      protocol: tcp
+      mode: host
+    volumes:
+    - type: bind
+      source: /mnt/c/Users/skunkworks/Projects/jempi/JeMPI/devops/windows/base-docker-wsl/docker_data/data/postgres
```

Comment:
Using local machine path.

Reply:
p.s this is also the same for all other services below

Reply:
this file should not be in the repo
it gets created by this line 
  envsubst < ./conf/stack/docker-stack-${SPEC_SETTINGS}.yml > ./docker-stack.yml
from deploy.sh in helper/scripts